backend/tracim_backend/applications/agenda/application.py

```diff
@@ -28,18 +30,26 @@ def load_config(self, app_config: CFG) -> None:
         app_config.CALDAV__RADICALE__STORAGE__FILESYSTEM_FOLDER = app_config.get_raw_config(
             "caldav.radicale.storage.filesystem_folder", default_caldav_storage_dir
         )
-        app_config.CALDAV__RADICALE__AGENDA_DIR = "agenda"
-        app_config.CALDAV__RADICALE__WORKSPACE_SUBDIR = "workspace"
-        app_config.CALDAV__RADICALE__USER_SUBDIR = "user"
-        app_config.CALDAV__RADICALE__BASE_PATH = "/{}/".format(
-            app_config.CALDAV__RADICALE__AGENDA_DIR
+
+        # INFO - G.M - 2021-12-07 - internal config param to configure installed agenda hierarchy
+        app_config.RADICALE__CALENDAR_DIR = "agenda"
```

Comment:
As the wording is still confusing , i do created a specific ticket to be sure about the meaning of agenda and calendar: https://github.com/tracim/tracim/issues/5241, this current code is ok if we consider both retrocompatibility and agenda as dav resource instead of "calendar", but clarification may be better.